include/test/battle.h

```diff
@@ -199,6 +199,17 @@
  * - Instead of player and opponent there is playerLeft, playerRight,
  *   opponentLeft, and opponentRight.
  *
+ * AI_BATTLE_TEST(name, results...) and AI_DOUBLE_BATTLE_TEST(name, results...)
```

Comment:
I'd call these `AI_SINGLE_BATTLE_TEST` and `AI_DOUBLE_BATTLE_TEST` for consistency with `SINGLE_BATTLE_TEST` and `DOUBLE_BATTLE_TEST`.